src/selection_type.py

```diff
@@ -0,0 +1,51 @@
+from talon import Context, Module
+from dataclasses import dataclass
+
+mod = Module()
+ctx = Context()
+
+ctx.matches = r"""
+app: vscode
+"""
+
+SELECTION_TYPE_KEY = "selectionType"
+
+
+@dataclass
+class SelectionType:
+    singular: str
+    plural: str
```

Comment:
Shall we remove plurals?  I've never used them and they're only useful for eg "lines in funk", but I think fine to just expect people to say "line funk".  Idk cc/ @knausj85 @brxck

Reply:
I personally don't use them.

Reply:
I don't use them either.